src/transformers/training_args.py

```diff
@@ -720,6 +725,16 @@ class TrainingArguments:
             )
         },
     )
+    save_safetensors: Optional[bool] = field(
+        default=None,
```

Comment:
I wouldn't pick an unset default, especially as we plan to phase in safetensors as a core dependency. It's going to be very surprising to users to have checkpoints suddenly saved in safetensors format. We will move forward (at some point in the future) to `save_pretrained` defaulting to safetensors, so I think this should be `False` for now, and we will switch to `True` at the same time as `save_pretrained`.

Reply:
Done